web/carousel/src/screens/Home/index.js

```diff
@@ -1,19 +1,70 @@
-import ProductTable from "../../components/Product/ProductTable";
+import ProductTable from "../../components/Product/ProductTable"
 
 export default function Home() {
   const productList = [
     {
       imageUrl:
         "https://store.storeimages.cdn-apple.com/4668/as-images.apple.com/is/mbp16touch-space-select-201911_GEO_TR?wid=892&hei=820&&qlt=80&.v=1582326712648",
       name: "Macbook Pro 16 inch",
-      price: 25000.0,
+      price: 999.99,
```

Comment:
25000 🆚 999
This makes me sad 😢

Reply:
Actually, it should be ~$2199 . I think it is better to change it, and then you may feel better :D

Reply:
😄 no need, thanks